index.js

```diff
@@ -159,6 +159,10 @@ class CameraRollPicker extends Component {
     fetchParams.include = [ "playableDuration" ]; // Include playableDuration by default as this will remove thumbnails that don't exist. e.g. /storage/emulated/0/ddmsrec.mp4 from screen recording.
 
     CameraRoll.getPhotos(fetchParams)
+      .then(data => {
+        data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp);
+        return data;
```

Comment:
Do we need to `return data` here explicitly? Or does `data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp)` return it automatically?

Reply:
@jarod-legault Try doing this and see if it works: 

```js
.then(data => data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp))
```

Reply:
Also, how about moving the error handler to a `catch` to be more explicit?

```js
CameraRoll.getPhotos(fetchParams)
      .then(data => data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp))
      .then(data => this.appendImages(data))
      .catch(e => console.log(e));
```

And we can combine the code in the `then`s

```js
CameraRoll.getPhotos(fetchParams)
      .then(data => {
        data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp);
        this.appendImages(data);
      })
      .catch(e => console.log(e));
```

Reply:
We can even make it smaller:

```js
CameraRoll.getPhotos(fetchParams)
      .then(data => this.appendImages(data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp)))
      .catch(e => console.log(e));
```

Reply:
I prefer the clarity in: 

```js
.then(data => { 
  data.edges.sort((a, b) => b.node.timestamp - a.node.timestamp); 
  this.appendImages(data); 
})
```

But double check that the `sort` function is "in-place" and that `data` is correctly sorted after running it. Otherwise, you may need to do `data = data.edges.sort( ... )`.

Reply:
Ok. I have updated the code to match your last suggestion and verified that `sort` is "in-place". Should be good to go.